luigi/file.py

```diff
@@ -35,6 +35,9 @@ class atomic_file(AtomicLocalFile):
     Also cleans up the temp file if close is not invoked
     """
 
+    def __init__(self, path, mode):
+        super(AtomicLocalFile, self).__init__(path, mode)
+
```

Comment:
@dlstadther probably; I will try. Could you recomment a place to add a unit test for this (roughly the code I pasted in description on top)? I'm not familiar with the Luigi codebase